src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.eventClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.eventClick);
+  }
+
+  eventClick = (event: KeyboardEvent) => {
+    this.setState({ key: event.key });
+  };
+
+  render() {
+    const { key } = this.state;
+
+    return (
+      <div className="App">
+        {
+          key
+            ? <p className="App__message">{`The last pressed key is [${key}]`}</p>
+            : <p className="App__message">Nothing was pressed yet</p>
+        }
```

Comment:
```suggestion
        {key
          ? <p className="App__message">{`The last pressed key is [${key}]`}</p>
          : <p className="App__message">Nothing was pressed yet</p>
        }
```

Reply:
the linter does not allow making a closing bracket on a new line